This fixes an error for an implicit definition for strerror().
launchd_fd.c: In function 'launchd_display_fd':
launchd_fd.c:60: warning: implicit declaration of function 'strerror'

kencu@macports.org


diff --git a/hw/xquartz/mach-startup/launchd_fd.c b/hw/xquartz/mach-startup/launchd_fd.c
index 15866cc42..a8b5efaf5 100644
--- a/hw/xquartz/mach-startup/launchd_fd.c
+++ b/hw/xquartz/mach-startup/launchd_fd.c
@@ -33,6 +33,7 @@
 #include <launch.h>
 #include <asl.h>
 #include <errno.h>
+#include <string.h>
 
 #include "launchd_fd.h"
 
